### DISCUSSION TITLE
Benefits not available despite being approved for Github education

### DISCUSSION THREAD

**Discussion by SidBthegr8** [community/community#138713] [GitHub Education]
### Hi everyone,

Hi all, I have been approved for the Github Education Pack as a verified student but haven't received the benefits despite it being 5 days now. Please help me expedite this process. Thanks.

**Comment by davevad93** [comment]
Hi @SidBthegr8 ,

GitHub no longer provides manual reviews and status updates. The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks. 

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @SidBthegr8 was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
